storage/cloud-client/hmac.py

```diff
@@ -29,14 +29,9 @@ def list_keys(project_id):
     hmac_keys = storage_client.list_hmac_keys(project_id=project_id)
     print('HMAC Keys:')
     for hmac_key in hmac_keys:
-        print('The HMAC key metadata is:')
-        print('Key ID: {}'.format(hmac_key.id))
+        print('Service Account Email: {}'.format(
```

Comment:
Email is missing from the other samples. I missed that metadata field in Ruby and saw it was missing here as well. Only additional ask I have.

Reply:
Added to the other samples. Let me know if you see anything else!